prometheus.go

```diff
@@ -29,7 +29,8 @@ type PrometheusCollector struct {
 
 func NewPrometheusCollector(metrics []*PrometheusMetric) *PrometheusCollector {
 	return &PrometheusCollector{
-		metrics: removeDuplicatedMetrics(metrics),
+		//metrics: removeDuplicatedMetrics(metrics),
```

Comment:
I am pretty sure that we still need this. It is used to ensure that if somebody finds a metric two times through different tags for example the exporter does not fall. But i will merge this and find a better logic for this use case.